lib/cordial/contacts.rb

```diff
@@ -58,7 +58,7 @@ def self.create(email:, attribute_list: {}, subscribe_status: nil)
           }.compact
         },
         forceSubscribe: subscribe_status == 'subscribed' || nil
-      }.compact.merge(attribute_list))
+      }.compact.merge(attribute_list).to_json)
```

Comment:
We have to do this now because of the added headers?

😢

Reply:
yes 😢